frontend/src/components/FundingContent.tsx

```diff
@@ -0,0 +1,56 @@
+import { nofoPdfs } from "src/constants/nofoPdfs";
+
+import { useTranslation } from "next-i18next";
+import { Grid, GridContainer } from "@trussworks/react-uswds";
+
+import PdfContainer from "./PdfContainer";
+
+const FundingContent = () => {
+  const { t } = useTranslation("common", { keyPrefix: "Index" });
+
+  return (
+    <div className="bg-base-lightest">
```

Comment:
Can we add some dynamic top/bottom padding to this section? LIke what @daphnegold did [on the hero section](https://github.com/HHS/grants-equity/pull/376/files#diff-65095e38dff2e5b202c1dbc07a9bc728483544c91a5524dbc2d59449de559af3R12): 

```
  className="bg-primary desktop:padding-y-15 tablet:padding-y-2 padding-y-1"
```

Not as much as "15" but this same technique.

Reply:
I noticed this in the Goal Content component, that the fonts are kinda wonky and we're swapping between serif and sans. Just an observation we may want to address now or in the future.

Reply:
sounds good, I added `className="desktop:padding-y-4 tablet:padding-y-2 padding-y-1"` to both the funding and goal content sections.